client/landing/stepper/declarative-flow/internals/steps-repository/goals/index.tsx

```diff
@@ -113,10 +111,7 @@ const GoalsStep: StepType< {
 
 	const recordNavigationSelectTracksEvent = ( intent: Onboard.SiteIntent, action: string ) => {
 		recordTracksEvent( 'calypso_signup_intent_select', { intent } );
-		recordTracksEvent( 'calypso_signup_goals_nav_click', {
-			action,
-			is_goals_first: isGoalsAtFrontExperiment,
-		} );
+		recordTracksEvent( 'calypso_signup_goals_nav_click', { action } );
```

Comment:
I also removed [this one here](https://github.com/Automattic/wp-calypso/pull/101864/commits/2f4b0ec3507f9e14079ba186c89b4d3a9a985298).